roles/openshift_health_checker/openshift_checks/memory_availability.py

```diff
@@ -0,0 +1,31 @@
+# pylint: disable=missing-docstring
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class MemoryAvailability(OpenShiftCheck):
+    """Check that recommended memory is available."""
+
+    name = "memory_availability"
+    tags = ["preflight"]
+
+    recommended_memory_mb = {
+        "nodes": 8000.0,
```

Comment:
This is memory though which isn't coming from docker info... comes from ansible setup. Whatever our docs say, nobody measures RAM in GB (only disk manufacturers get to pull that crap), so it seems to me MiB/GiB should be used.